builder/docs_builder.livecodescript

```diff
@@ -1970,11 +1976,7 @@ private command docsBuilderGetDictionaryFolders pEdition, @xFolders
    end if
    
    if pEdition is "business" then
-      local tBusinessDocs
-      put RepoBusinessDocsFolder() into tBusinessDocs
-      if there is a folder tBusinessDocs then
-         addToList tBusinessDocs, xFolders
-      end if
+      addFolderToListIfExists RepoBusinessDocsFolder(), xFolders
```

Comment:
How about adding the Indy edition docs folder using `addFolderToListIfExists` too?